### DISCUSSION TITLE
Demo lenis integration in HTML/CSS

### DISCUSSION THREAD

**Discussion by aleco88** [darkroomengineering/lenis#205] [Q&A]
Hi, thanks for sharing this amazing lib!

I was wondering if there's any chance to have this lib. with examples and files shared with plain HTML files, instead of next.js?

Thanks, and congrats again for the amazing work!

**Comment by fabricio-fn** [comment]
I don't think so, friend, see the examples in the next.js documentation https://nextjs.org/docs
